kernel/README.md

```diff
@@ -24,15 +24,87 @@ in the kernel source code. You can modify it as needed.
 After the kernel source code is ready it is possible to build the kernel.
 
 ```bash
-./build-kernel.sh build
+$ ./build-kernel.sh build
 ```
 
 
 ## Install the Kernel in the default path for Kata
 
 Kata Containers uses some default path to search a kernel to boot. To install
 on this path, the following command will install it to the default Kata
-containers path.
+containers path.  
+
 ```bash
-./build-kernel.sh install
+$ ./build-kernel.sh install
+```
+
+## Summit Kernel Changes
+
+Kata Containers packaging repository holds the kernel configs and patches. The
+config and patches could be for many versions but we only make sure to test the
+kernel version defined on the [runtime versions file][runtime-versions-file]
+
+## How is it tested.
+
+Kata Containers CI scripts will install the kernel from packages or builder
+from sources.
+
+If kernel defined in [runtime versions file][runtime-versions-file] is build
+with latest kernel config and patches then patches are installed. In other case
+the kernel is build from source.
+
+To identify if the packaged kernel is up-to-date the package kernel package
+version is a concatenation from the kernel version defined [runtime versions
+file][runtime-versions-file]
+
+exmaple:
+
+``` # from https://github.com/kata-containers/runtime/blob/master/versions.yaml
+kernel_version_in_versions_file=4.10.1 #from
+https://github.com/kata-containers/packaging/blob/master/kernel/kata_config_version
+kernel_config_version_form_kata=25
+
+latest_kernel_version=${kernel_version_in_versions_file}.${kata_config_version}
 ```
+
+The resulting version will be `4.10.1.25`, this help to identify if the kernel
+configs are up-to-date on a package version.
+
+# Contribute
+
+In order to do Kata Kernel changes . There are places to contribute:
+
+1. [Kata runtime versions file][runtime-versions-file]: This file points to the
+   recommended to use by Kata. To update the kernel version send a pull request
+   to update that version. The Kata CI will run all the use cases and verify it
+   works.
+
+1. Kata packaging repository. This repository contains all the kernel configs
+   and patches recommend for the Kata kernel: 
+
+- If you want to upload one new configuration (new version or architecture
+  especific) make sure the config file name has the following format:
+
+```bash
+# Format:
+$ ${arch}_kata_${hypervisor_target}_${major_kernel_version}.x 
+# example:
```

Comment:
I'd add a blank line before this comment line for clarity (same comment for the comment below :)